<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QWidget" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChatRoom</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color:#1e2833;
}

QPushButton {
  background-color:#214a80;
  border:none;
  border-radius:4px;
}

QPushButton:hover{
background-color:  rgb(240, 62, 140);
}

QListWidget, QLineEdit {
border: none;
border-radius:4px;
background-color: #233142;
}

QListWidget::item {
height: 30px;
selection-background-color: #1e2833;
}

QListWidget::item:active {
border:none;
}

QListWidget::item:selected {
border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="8,1" columnstretch="5,2" rowminimumheight="0,0" columnminimumwidth="2,1">
   <item row="0" column="0">
    <widget class="QListWidget" name="msgboxWidget">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="userlistWidget">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="msginputLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>enter msg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="sendmsgButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
